README.md

```diff
@@ -29,7 +29,7 @@ This project has been
 
 From the root of the source tree, run:
 
-    make generate all
+    TAGS=bindata make generate all
```

Comment:
my opinion is that this change should remain as is because bindata is only optional, and shouldn't be enforced.

Reply:
@techknowlogick I would prefer to have it as all releases we provide are with it and it makes so much more problems if it is built without bindata

Reply:
ok. sounds good. LGTM.